packages/app/src/store/studio-store.ts

```diff
@@ -853,7 +864,7 @@ export const useStudioStore = defineStore('studioRecorder', {
       return $el.hasClass('__cypress-studio-assertions-menu')
     },
 
-    _openAssertionsMenu (event) {
+    _openAssertionsMenu (event, addAssertion?: ($el: HTMLElement | JQuery<HTMLElement>, ...args: AssertionArgs) => void) {
```

Comment:
adding the ability to pass in `addAssertion` here allows us to control assertion recording from the cloud without having to move the whole assertion menu UI over - see the other PR for how we hook into this